lib/msf/core/auxiliary/masscan.rb

```diff
@@ -0,0 +1,249 @@
+# -*- coding: binary -*-
+require 'rex/parser/masscan_nokogiri'
+require 'rex/parser/masscan_xml'
+require 'open3'
+
+module Msf
+
+###
+#
+# This module provides methods for interacting with masscan.
+# Modules that include this should define their own masscan_build_args()
+# function, and usually should have some method for dealing with
+# the data yielded from masscan_hosts(). See auxiliary/scanner/oracle/oracle_login
+# for an example implementation.
+#
+###
+
+module Auxiliary::Masscan
```

Comment:
This seems to be completely unused.  I am guessing this was added given the similar code for nmap exists, but at least right now there is not a use-case defined for this.  I'll just remove it, saving cleanup.